### PR TITLE
feat(alerts): New alerts alerts stream and rules list header

### PR DESCRIPTION
**Pull request by scttcper**
Matches our other newer pages like discover.

image

image

### CODE REVIEW

File: docs-ui/components/layout-thirds.stories.js [hunk 1]
```diff
@@ -137,7 +137,7 @@ export const SingleColumnMode = withInfo('Single column mode so we can hide the
           </ButtonBar>
         </Layout.HeaderActions>
       </Layout.Header>
-      <Layout.Body fullWidth>
```
Comment (scttcper) [inline docs-ui/components/layout-thirds.stories.js:140]: updated storybook, fullwidth only exists on `layout.main`

File: src/sentry/static/sentry/app/views/alerts/list/header.tsx [hunk 2]
```diff
@@ -47,45 +46,49 @@ const AlertHeader = ({router, organization, activeTab}: Props) => {
...
+`;
+
+const StyledLayoutTitle = styled(Layout.Title)`
+  margin-top: 0;
 `;
 
 const StyledNavTabs = styled(NavTabs)`
+  margin-top: 15px;
+  margin-bottom: 0;
+  border-bottom: 0 !important;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/alerts/list/header.tsx:91]: Why do we zero out these margins if we're trying to match Discover's layout

**Review by billyvg — COMMENTED** [review]
The Create Alert Rule button looks a bit *too* big and is it vertically centered with the Alerts title?

File: src/sentry/static/sentry/app/views/alerts/list/header.tsx (hunk 2 above)
Comment (scttcper) [reply src/sentry/static/sentry/app/views/alerts/list/header.tsx:91]: discover isn't using a tab navigation here, they have a title instead.

Comment (robinrendle) [reply src/sentry/static/sentry/app/views/alerts/list/header.tsx:91]: Yeah, we should build tabs into the Layout component eventually so we don’t have to do all this styled stuff.

**Comment by scttcper** [comment]
@billyvg @robinrendle had me not vertically center with the Alerts title for now with a plan of supporting larger text there at the layout level soon. Same with the somewhat messy nav tabs removing the margin-bottoms, we want to add a tabbed navigation that fits into the `Layout` without modification that also works on mobile.

We could make the action buttons smaller for now?

**Review by billyvg — APPROVED** [review]

**Comment by billyvg** [comment]
I'll leave design up to you and @robinrendle -- imo they need to be a bit smaller, but the code looks good

**Pull request merged**